packages/form-builder/src/components/onboarding/steps/index.tsx

```diff
@@ -0,0 +1,113 @@
+import {__} from "@wordpress/i18n";
+import {compose} from "@wordpress/compose";
+import withButtons from "./withButtons";
+import withText from "./withText";
+import withDefaults from "./withDefaults";
+
+type Placement = 'top'|'top-start'|'top-end'|'bottom'|'bottom-start'|'bottom-end'|'right'|'right-start'|'right-end'|'left'|'left-start'|'left-end';
+
+export default Object.values(compose(
+    withText,
+    withButtons,
+    withDefaults({
+        canClickTarget: false,
+        scrollTo: false,
+        cancelIcon: {
+            enabled: false,
+        },
+        arrow: false,
+    }),
+)([
+    {
+        id: 'welcome',
+        title: __('Welcome to the visual donation form builder!', 'give'),
+        text: __('The following is a quick (less than a minute) tour of the visual donation form builder, to introduce the tools for creating engaging donation forms.', 'give'),
+    },
+    {
+        id: 'canvas',
+        attachTo: { element: '#FormBlocksCanvas', on: 'right-start' as Placement },
```

Comment:
I noticed the Id's are being added as camelcase which always feels weird to me as standard convention is typically `form-blocks-canvas` for the ID attribute

Reply:
I always thought that the convention was camleCase for IDs and kebab-case for classes.

Reply:
Technically both is valid, I personally prefer the latter & if camelCase the first letter should be lowercase - but looking at how WP they are using kebab

<img width="1050" alt="Screenshot 2023-03-31 at 1 27 04 PM" src="https://user-images.githubusercontent.com/10138447/229189098-8cf5c615-fdad-4e0d-ae95-feddff54d1c8.png">

Reply:
Resolved, see https://github.com/impress-org/givewp-next-gen/pull/143/commits/3b33f9cbb341fba69412e53d6943eee70d8ec77d.